crawlers/ten_ago_dbcnt.py

```
@@ -0,0 +1,59 @@
+import sys
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)